applications/nrf_desktop/Kconfig

```diff
@@ -16,5 +16,9 @@ rsource "Kconfig.defaults"
 endmenu
 
 menu "Zephyr Kernel"
+# RAM for this application for these boards is too few to use
+# the BT TX processor thread.
+configdefault BT_TX_PROCESSOR_THREAD
```

Comment:
btw. Could we introduce a prompt for `CONFIG_BT_TX_PROCESSOR_THREAD` upstream to simplify enabling it for small applications? (reducing memory usage might be relevant for some use-cases)

Reply:
User enabling it explicitly would be aware of potential issues/limitations (we could e.g. describe them in Kconfig help)